picard/script.py

```diff
@@ -413,6 +413,13 @@ def func_unset(parser, name):
     """Unsets the variable ``name``."""
     if name.startswith("_"):
         name = "~" + name[1:]
+    # Allow wild-card unset for certain keys
+    if name in ('performer:*', 'comment:*', 'lyrics:*'):
```

Comment:
I wonder if we could make this even more generic, `anytag:*` could be used to delete all `anytag`.